controllers/nnf_node_ec_data_controller.go

```diff
@@ -83,8 +86,19 @@ func (r *NnfNodeECDataReconciler) Start(ctx context.Context) error {
 		persistent.StorageProvider = r
 	}
 
+	// NNF Element Controller logger provides 4 verbosity levels. By default, nnf-ec
+	// will use the raw logger provided. If you are using zap defaults, this will
+	// only include the INFO and DEBUG levels. You can increase the defaults, by
+	// manually setting the zap.Options.Level, or using the -zap-log-level flag.
+	//
+	// You can decrease the log level of nnf-ec by adding verbosity levels to passed
```

Comment:
Should this be, "by adding a verbosity level that is passed into the logger"?

Reply:
I rewrote that line - please take a look and see if it makes sense this time around